pkg/artifactory/resource_artifactory_local_repository.go

```diff
@@ -143,72 +140,88 @@ func resourceArtifactoryLocalRepository() *schema.Resource {
 	}
 }
 
-func unmarshalLocalRepository(s *schema.ResourceData) *v1.LocalRepository {
-	d := &ResourceData{s}
-
-	repo := new(v1.LocalRepository)
-
-	repo.RClass = artifactory.String("local")
+type MessyRepo struct {
```

Comment:
I like this name ;-D

Reply:
I will just create a ticket to address this and fix it right away